pages/installation.md

```diff
@@ -0,0 +1,49 @@
+# The program is shipped in one executable file that acts both as the server and the client.
+
+Acquire it as below for your system:
+
+## Linux amd64
+```bash
+curl -LO https://github.com/boringproxy/boringproxy/releases/download/v0.5.0/boringproxy-linux-amd64d
```

Comment:
Can you change v0.5.0 to the latest release? That way we won't have to update all of them every time there's a new release.

Info on linking to releases:

https://docs.github.com/en/github/administering-a-repository/linking-to-releases

Reply:
I don't think you can do that until 1.0. I get a 404 from <https://github.com/boringproxy/boringproxy/releases/latest/download/boringproxy-linux-amd64> , and no latest release tag shows up on the releases page. If you know how, please tell me, but I am leaving it for now.